packages/wallet-frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "wallet-frontend",
   "scripts": {
     "dev": "PORT=4003 next dev",
-    "build": "next build",
+    "build": "NEXT_PUBLIC_BACKEND_URL=http://35.196.11.156:3003 next build",
```

Comment:
is this change required?

Reply:
well, the env values in docker-compose.yml wasn't affecting the build environment variables.
Which means, without this, that backend url was setting to another value.

Reply:
I also find it a bit weird to add the env variables here in the build script